src/annotator/test/highlighter-test.js

```diff
@@ -460,6 +460,47 @@ describe('annotator/highlighter', () => {
 
       assert.deepEqual(orderedNestingLevels, [0, 0, 0, 0, 1, 2]);
     });
+
+    it('orders focused SVG highlights last regardless of nesting level', () => {
+      const container = document.createElement('div');
+      render(<PDFPage />, container);
+
+      const svgEls = () => Array.from(container.querySelectorAll('rect'));
+      const orderedNestingLevels = () => svgEls().map(el => nestingLevel(el));
+
+      // This highlight has a nesting level of 0
+      const toFocus = highlightPDFRange(container, 'user-annotations');
+      highlightPDFRange(container, 'user-annotations'); // Nesting level 1
+      highlightPDFRange(container, 'other-content'); // Nesting level 2
+
+      // Initial nesting-based ordering
+      updateClusters(container);
+
+      assert.equal(svgEls().length, 3);
+      assert.deepEqual(orderedNestingLevels(), [0, 1, 2]);
+
+      // Focus the first, outermost highlight
+      setHighlightsFocused(toFocus, true);
```

Comment:
It would be useful to have the tests verify that `setHighlightsFocused` is a no-op if the `focused` argument is the same as the current state.

Reply:
Done (on working branch).